pkg/logs/agent_not_serverless.go

```diff
@@ -0,0 +1,125 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
NIT: I think keeping this file called `agent.go` and the other one `agent_serverless.go` is more consistent with the rest of our code base. WDYT?

Reply:
Completely agree, problem is there's another file in the package already named `agent.go`. Any ideas of what else either that file or this file could be named? I didn't come up with anything earlier.

Reply:
Oh 🤦 I totally missed that file wasn't deleted. 
Maybe `agent_init.go`? I see there is at least one other place we do `_not_serverless.go` - so I guess it's fine as is too. Ill leave it up to you.